frontend/src/components/QuickAdd.tsx

```diff
@@ -79,27 +73,45 @@ export const QuickAdd = ({ addIssueActivity }) => {
     setActivity(activity);
   };
 
+  const getSearchClasses = () => {
+    let classes = "col-2 quick-add-input ";
+    if (search != "") classes += issue ? "valid" : "invalid";
+    return classes;
+  };
+
+  const getValidationIconSrc = () => {
+    let src = "";
+    if (search != "") src = issue ? check : x;
+    return src;
+  };
+
   return (
     <div className="row">
       <h2>Quick add:</h2>
-
       <input
         aria-label="Issue"
         id="input-issue"
-        className={classes}
+        className={getSearchClasses()}
         type="number"
         min={0}
-        onChange={(e) => setSearch(e.target.value)}
+        onChange={(e) => {
+          setSearch(e.target.value);
+          setIssue(undefined);
+        }}
         placeholder="Type issue number..."
         title={(issue && issue.subject) || ""}
       />
+      <img
+        className={search == "" ? "validation-icon hiden" : "validation-icon"}
+        src={getValidationIconSrc()}
+      />
       <select
         aria-label="Activity"
         className="col-3"
         name="activity"
         id="select-activity"
         onChange={handleSetActivity}
-        disabled={classes.includes("invalid")}
+        disabled={search != "" && issue == undefined}
```

Comment:
another `==` that should be a `===`

Reply:
not really same as above the first one and issue == undefined does not need to convert. You cant convert an undefined value